earth_enterprise/src/server/wsgi/serve/publish/publish_manager_helper.py

```diff
@@ -1213,7 +1213,7 @@ def _GetPortableGlobesList(self, registered_portable_set):
     root = constants.CUTTER_GLOBES_PATH
     for name in os.listdir(root):
       # Ignore globes that are registered.
-      if (name not in registered_portable_set) and (os.path.isfile(name)):
+      if name not in registered_portable_set and os.path.isfile(os.path.join(root,name)):
```

Comment:
so was the cause of that error "no portable..." the old code that use short file name instead of the correct file name (full path) ? Basically the globe was there but we did not find because of it's short name ?

Reply:
Yeah.  The previous check fixed the symlink problem because almost everything would fail the isfile check but that started preventing valid portables from being built.  Not sure what directory the server is executing in but it's apparently not /opt/google/gehttpd/htdocs/cutter/globes

Reply:
Thanks. This is an introduced regression issue when fixing #252 ? (https://github.com/google/earthenterprise/commit/26081a0f8ebabaeb6343bafbeb404c6268c98e0e). Wanted to know so we would not document it an release notes for 5.2.2 if it was a introduced then fixed issue.

Reply:
I believe that is correct.  Before the fix for 252 managing portables looks like it worked fine.  As long as this is fixed there should be no change in behavior.